File: Praktikum5/ProgramC/tempCodeRunnerFile.c
#include <stdio.h>

void Biodata(int tahunLahir, char nama[], char asal[]) {
    int tahunSekarang = 2020;
    int umur = tahunSekarang - tahunLahir;

    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahunSekarang);
    printf("Asal Saya dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];

    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", Namaku);
    scanf(" %[^\n]%*c", Asal);

    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}

File: Praktikum5/ProgramPython/tempCodeRunnerFile.py
def Biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir

    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")


tahun_lahir = int(input())
nama = input()
asal = input()

Biodata(tahun_lahir, nama, asal)
